### PR TITLE
feat: responsive docsite navigation added.

### PR DESCRIPTION
**Pull request by krishnakant01**
I've added a responsive navbar with a hamburger menu. Please ignore the practice.html, practice.css, and app.js files.
Resize your viewport in deploy preview to see changes.

### CODE REVIEW

**Review by harshalini — APPROVED** [review]
Code is nice, you can merge it

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -2,64 +2,192 @@
...
     background-color: var(--color-wheat);
-    }
-.nav-container-bg-gradient{
+}
+
+.nav-container-bg-gradient {
     background-image: linear-gradient(45deg, var(--color-primary-complimentary), var(--color-medium-blue));
 }
+
+.sr-only {
```
Comment (t007rushi) [inline components/navigation/navigation.css:18]: seems generic you can add it to utility class

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -2,64 +2,192 @@
...
     font-weight: 500;
-    color: var(--color-darker-gray);;
+    color: var(--color-darker-gray);
+
 }
+
 .nav-tagline {
     font-weight: 500;
     font-size: 1rem;
     color: #1f1f1f;
```
Comment (t007rushi) [inline components/navigation/navigation.css:41]: use color variables here

File: components/navigation/navigation.css [hunk 3]
```diff
@@ -68,22 +196,27 @@
...
     color: var(--color-white);
     text-decoration: none;
     padding: 0.5rem;
     margin-right: 0.5rem;
     position: relative;
 }
-.navigation-container .link::after{
+
+.navigation-container .link::after {
     content: "";
```
Comment (t007rushi) [inline components/navigation/navigation.css:220]: empty prop?

File: practice.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    font-size: 2rem;
+    font-weight: 500;
+    color: var(--color-darker-gray);
+
+}
+
+.nav-tagline {
+    font-weight: 500;
+    font-size: 1rem;
+    color: #1f1f1f;
```
Comment (t007rushi) [inline practice.css:41]: same as above

File: practice.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (t007rushi) [inline practice.html:8]: change it to page name

**Review by t007rushi — CHANGES_REQUESTED** [review]

**Pull request merged**